core/polyaxon/client/client.py

```diff
@@ -62,9 +62,9 @@ class PolyaxonClient:
     def __init__(self, config=None, token=None):
 
         self._config = config or settings.CLIENT_CONFIG
-        if not token:
```

Comment:
You can use:
```python
token = token or self._config.token
```
and keep the previous logic intact, that was initially what I was going to do for the fix.

Reply:
Done

Reply:
The problem is that with current logic, user can't override token with 'no-auth' to remove token set in config.

Reply:
We need to force `None` now on the config token [value](https://github.com/polyaxon/polyaxon/pull/1365/files#diff-ac79b6e7d8c6f3c00aec459aff938f4ee1fd4a5d2b9b980da61268230da4ec38R69):

```python
if not token:
    self._config.token = settings.AUTH_CONFIG.token
elif token == NO_AUTH:
    self._config.token = None
else
    self._config.token = token
```

Reply:
Fixed